docs/tutorials/services/visualize-data-grafana.md

```diff
@@ -104,7 +104,7 @@ To enable data capture for a sensor component:
 1. Click **Save Config** at the bottom of the window to save your changes.
 
 After a short while, your sensor will begin capturing live readings, and syncing those readings to the Viam app.
-To verify this, ensure that you are able to [view your data in the Viam app](/data/view/).
+You can check that data is being captured by clicking on the menu on the sensor and selecting **View captured data**.
```

Comment:
same opt nit here:

```suggestion
You can check that data is being captured by clicking on the menu icon on the sensor configuration pane. and selecting **View captured data**.
```

For these, the `icon` can be skipped, but we are clicking on a UI element _for_ (representing) the camera | sensor, and not that resource itself. I feel that `configuration pane` or at least `pane` helps to clarify things like (in the web UI, under the **CONFIGURE** tab, in the Viam App, etc).

But they do all have screenshots accompanying them, so it's likely fairly evident anyways. Feel free to ignore if you like.